keyboards/splitkb/zima/readme.md

```diff
@@ -0,0 +1,15 @@
+# Zima
+
+![Zima](https://cdn.shopify.com/s/files/1/0227/9171/6941/products/Zima_Clear_Side.jpg?v=1596531772)
+
+A feature-packed twelve key macropad the size of a creditcard.
+
+* Keyboard Maintainer: [Thomas Baart](https://github.com/splitkb)
+* Hardware Supported: Onboard ATmega32u4 with speaker, encoder, RGB underglow, OLED and haptic support, with a Type C connector
+* Hardware Availability: [splitkb.com](https://splitkb.com/collections/keyboard-kits/products/zima)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make splitkb/zima:default
+
```

Comment:
Gib "clear instructions on how to reset the board into bootloader mode" pls.

Reply:
There's a reset button on the board.

Reply:
> Gib "clear instructions on how to reset the board into bootloader mode" pls.

Not related to this PR, but we ... should probably add that to the template readme file.